crypto/libsodium-fork/src/libsodium/crypto_aead/aes256gcm/aesni/aead_aes256gcm_aesni.c

```diff
@@ -365,7 +365,7 @@ do { \
   */ \
     MAKE4(RED_MUL_MID); \
 \
-/* substracts x1*h1 and x0*h0 */ \
+/* subtracts x1*h1 and x0*h0 */ \
```

Comment:
this is copy from libsodium, maybe we should not touch it at all :)

Reply:
I think that this is going to be deprecated along with the rest of the libsodium.. so I'm not particular picky around this one. But you're correct that it's not a "typo" that we've "generated" on our own. Maybe we should recommend libosdium to use this tool as part of their build toolchain.